python/lsst/afw/display/interface.py

```diff
@@ -523,27 +526,26 @@ def pan(self, colc=None, rowc=None, origin=afwImage.PARENT):
         """!Pan to (rowc, colc); see also zoom"""
 
         self.zoom(None, colc, rowc, origin)
-
+    
```

Comment:
Yes, we try to remove trailing whitespace. We are also actively updating the style guide to be able to express it in terms of PEP-8.